ClassLibrary1/Static_001Helper.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Linq;
+namespace ClassLibrary1
+{
+    public class Static_001Helper
+    {
+        public static double FirstNum { get; set; }
+        public static double SecondNum { get; set; }
+        public static double Sum() => FirstNum + SecondNum;
+        public static double Sub() => FirstNum - SecondNum;
+        public static double Mult() => FirstNum * SecondNum;
+        public static double Div()
+        {
+            if (SecondNum == 0)
```

Comment:
You should invert conditions here because `== 0` will occurs rarely than defalut dividing.